docs/apm/traces/working-with-tracing-data/service-map.md

```diff
@@ -41,7 +51,7 @@ Manual anomaly detection settings allow you to specify different detection setti
 
 ![service map manual detection settings.png](/img/traces/service-map-manual-detection-settings.png)
 
-## Service Map Dashboard Panels
+### Service Map Dashboard Panels
```

Comment:
The dashboard panel is called "Services" now with additional option to configure service list OR map as chart type. Please update the content and screenshots.